packages/element/index.ts

```diff
@@ -124,4 +124,4 @@ export { suite } from '@flood/element-core'
 /**
  * @docPage helpers
  */
-export { RecoverWith } from '@flood/element-core'
+export { RecoverWith, SummaryStep } from '@flood/element-core'
```

Comment:
correct me if I'm wrong, `SummaryStep` is used internally to build test result, so I think we don't need to expose it to users
```suggestion
export { RecoverWith } from '@flood/element-core'
```